examples/metering.rs

```diff
@@ -21,7 +21,7 @@ use wasmer::CompilerConfig;
 use wasmer::{imports, wat2wasm, Instance, Module, Store};
 use wasmer_compiler_cranelift::Cranelift;
 use wasmer_engine_jit::JIT;
-use wasmer_middlewares::Metering;
+use wasmer_middlewares::{get_remaining_points, set_remaining_points, Metering};
```

Comment:
Can we have the `get_remaining_points`, `set_remaining_points` only in `metering`?

I'm afraid in the future, when we'll have more middlewares that we have either conflicts os `use` satatements looking like 
```
use wasmer_middlewares::{get_remaining_points, set_remaining_points, some_middleware_func, other-middleware_func, yet_another_middleware_func, Metering};
```

Having something like 

```
use wasmer_middlewares::{
    metering::{get_remaining_points, set_remaining_points}, 
    Metering,
};
```

I think showing this as an example seems cleaner (to me at least, but I may be wrong).

Reply:
Nice idea! As you can see in the PR description I struggeled with namespacing of those. I did not realize the `metering` module is public already. Will update.

Otherwise good?

Reply:
Updated in the latest commit